tasks/main.yml

```diff
@@ -32,15 +32,13 @@
 
 - name: install riak.conf with templated configuration
   template: "src={{ riak_conf_template }} dest=/etc/riak/riak.conf owner=root group=root mode=0444 backup=yes"
-  notify: restart riak
```

Comment:
When you use notify, it waits till the playbook is done to run the Riak restart handler. We want it done sooner since we run tests against Riak for the smoke testing. This change should be transparent to end users but help us ensure proper config for smoke test runs.